### DISCUSSION TITLE
The language and framing of post modernity and meta modernity

### DISCUSSION THREAD

**Discussion by matthewmccarthy11** [life-itself/community#973] [Research]
Following on post #972 

It has been discussed that post modernity and meta modernity are both concerned with 'systems change'. And so, it might be interesting to consider 'post modern vocabulary' and 'meta modern vocabulary' or 'framing', if there is a difference. It has been discussed in the cohere project that 'regenerative' or 'well-being' can at times be more post modern than meta modern, as an example. Maybe 'post modern vocabulary' is more concerned with 'advertising', but also worth emphasizing that all orgs try to attract a certain audience.

What would meta modern vocabulary be, if there is one? Perhaps renaissance, radical, evolution, conscious cultural evolution. 

Mapping the different languages and 'framing' can be a way to consider the turning of the paradigms.

**Comment by asimong** [comment]
I'd just like to add into the mix the framing from Robert Kegan. In "In Over Our Heads" he separates postmodernism into two: the first, a transitional stage, (maybe 4.5?) he calls "deconstructive postmodernism". What he is valuing, and labelling as his 5th order consciousness, he calls "reconstructive postmodernism" (this is from memory, please correct me if I have misremembered.) I do think that is helpful in giving a positive as well as negative aspect to postmodern. However, I guess when people say "metamodern" they are talking about pretty much the same thing as reconstructive postmodernism. I'd like to have other readings on that — do you share my view of putting them together, or do you see a difference?

**Reply by rufuspollock** [reply]
@asimong i think the point that a given major "stage" probably has early, mature and late variants. And we should be careful of identifying "green/post-modern" with an early "reactive" version of it.

For example, one can easily point to early postmodern as having a tendency to deconstructive relativistic nihilism. However, it is clear that later "postmodern" has a return to values e.g. ecology, belonging, spirituality etc. (See Avatar vs Taxi Driver -- not arguing her for artistic merits but moral framing).

This is something that i'm coming to think more and more.

**Reply by asimong** [reply]
Fair enough, @rufuspollock but in that case what is the need to name a stage as "metamodern"? Or are you with me in drawing a rough equivalence between metamodern and Kegan's reconstructive postmodernism -- a mature variant?

**Reply by rufuspollock** [reply]
I was more saying that I would then reserve metamodern for something beyond this, or at least do some serious work to tease apart what distinguishes metamodern from postmodern.

**Reply by asimong** [reply]
nice challenge. How do you suggest proceeding? My sense includes the recognition that it might well be useful to include reckoning with the question, why metamodern? What is the role (if I may use that word very broadly) for those with metamodern sensibility in this contemporary world?

**Reply by asimong** [reply]
found a nice article about modern etc architecture. Quite interesting really and potentially relevant to our conversation here. https://aeon.co/essays/the-architectural-style-wars-have-started-all-over-again
